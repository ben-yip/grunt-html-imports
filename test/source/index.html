<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>Index</title>
</head>
<body>

<link rel="import" href="common/_header.html">

<link rel="import" href="component/_sidebar.html">

<p> Index page content </p>

<link rel="import" href="common/_footer.html">

</body>
</html>