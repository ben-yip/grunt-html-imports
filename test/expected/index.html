<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>Index</title>
</head>
<body>

<header>
    This is header.
</header>

<aside>
    this is sidebar.
</aside>

<p> Index page content </p>

<footer>
    This is footer.
</footer>

</body>
</html>